### ISSUE TITLE
Add a live image grab function 

### ISSUE THREAD

**Issue by gabs1234** [ufo-kit/libuca#96]
**This function is specifically needed for the Phantom camera:**
When triggered, it will save n_preframes and n_postframes in a partition of its ram (cine). During this aquisition time, our Phantom model cannot transmit the images it has just captured. It can only transfer them once the entire aquisition is finished.

It does however has a special partition called the live image partition from which we can grab the latest image seen by the phantom. By adding a live image function `uca_camera_grab_live()` in libuca, we could have two seperate memory spaces to grab images from.
 1. One for the normal aquisitions, sent over 10Gb
 2. One for the live images, sent over 1Gb

Without two seperate memroy spaces, we would have no way of knowing if an image came from the live cine or the triggered cine.

**Comment by gabs1234** [comment]
See pull request #97

**Comment by gabs1234** [comment]
No longer needed, see comment on pull request #97
